src/pages/index.tsx

```diff
@@ -1,87 +1,76 @@
-        <Container maxWidth="1185px">
-          <Heading size="md" fontWeight="500">Internet is essential</Heading>
-          <SimpleGrid columns={[1, null, 3]} spacing={[5, 10]} mt={['24px', '64px']}>
-            <Box>
-              <Heading color="purple" size="2xl" p={2}>79%</Heading>
-              <p>of Americans used internet for their last job search</p>
-            </Box>
-            <Box>
-              <Heading color="orange" size="2xl" p={2}>50%</Heading>
-              <p>of students say they need internet access to do their homework</p>
-            </Box>
-            <Box>
-              <Heading color="blue" size="2xl" p={2}>86%</Heading>
-              <p>of Americans get news from digital devices</p>
-            </Box>
-          </SimpleGrid>
-        </Container>
+      <Box bgColor="offWhite" w="100%"  textAlign="center" p={["24px", "64px"]}>
+        <Heading fontSize={24} fontWeight={500}>Internet is essential</Heading>
+        <Wrap spacing={["72px", "160px"]} mt={['24px', '64px']} justify="center">
+          <VStack spacing={"32px"} w={256}>
+            <Heading color="purple" fontSize={64} fontWeight={900}>79%</Heading>
+            <Text>of Americans used internet for their last job search</Text>
+          </VStack>
+          <VStack spacing={"32px"} w={256}>
+            <Heading color="orange" fontSize={64} fontWeight={900}>50%</Heading>
+            <Text>of students say they need internet access to do their homework</Text>
+          </VStack>
+          <VStack spacing={"32px"} w={256}>
+            <Heading color="blue" fontSize={64} fontWeight={900}>86%</Heading>
+            <Text>of Americans get news from digital devices</Text>
+          </VStack>
+        </Wrap>
       </Box>
-      <Box>
-        <Container maxWidth="1185px" p={['24px', '64px']} w="100%" textAlign="center">
-          <Stack spacing={10}>
-            <Heading size="md" fontWeight="500">Internet access is slow, expensive and segregated</Heading>
-            <iframe src="https://new-york-civil-liberties-union.github.io/NYC-Broadband-Map/" width="100%" height="700px" />
-            <Text>Only 46% of households living in poverty have internet access. 30% of Black households and 32% of Hispanic households lack home broadband vs. 20% of White households.</Text>
-          </Stack>
-        </Container>
-      </Box>
-      <Center bgColor="purple" p={['24px', '64px']} color="white" w="100%">
-        <Stack spacing={5}>
-          <Text align="center">Join us in our  campaign to make accessible, affordable public broadband a reality.</Text>
-          <Center>
-            <PrimaryButton
-              title="Take Action"
-              href="/take-action"
-              background="aqua"
-            />
-          </Center>
-        </Stack>
-      </Center>
+      <VStack spacing={["32px", "56px"]} p={["24px", "64px"]} textAlign="center">
```

Comment:
At lot of these changes are just removing unnecessary parent components and using VStacks